python/lsst/pex/config/config.py

```diff
@@ -660,6 +661,30 @@ def toDict(self):
             dict_[name] = field.toDict(self)
         return dict_
 
+    def names(self):
+        """!Return all the keys in a config recursively
+        """
+        #
+        # Rather than sort out the recursion all over again use the pre-existing saveToStream()
```

Comment:
Block comments have a maximum length of 79 characters. I don't like it, but them's the rules.